client/src/partials/patient_invoice/registry/registry.js

```diff
@@ -130,7 +132,18 @@ function InvoiceRegistryController(Invoices, Notify, Session, util, Receipt, App
     load(vm.filters);
   }
 
+  // Function for Credit Note cancel all Invoice
+  function creditNote(invoice) {
+    Invoices.openCreditNoteModal(invoice)
+      .then(function (data) {
+        // no parameters means the modal was dismissed.
```

Comment:
I don't think that's true.  You call `$uibModalInstance.dismiss()` which will reject the modal promise.  If the modal is dismissed, this function is never called.